docs/actions.md

```diff
@@ -1219,6 +1184,71 @@ This command starts a polling loop that continuously checks for logs from activa
 
   Similarly, whenever you run the poll utility, you see in real time the logs for any actions running on your behalf in OpenWhisk.
 
+## Getting actions
+
+Metadata that describes existing actions can be retrieved via the `wsk action get` command.
+
+```
+wsk action get hello
+ok: got action hello
+{
+    "namespace": "user@email.com",
+    "name": "hello",
+    "version": "0.0.1",
+    "exec": {
+        "kind": "nodejs:6",
+        "binary": false
+    },
+    "annotations": [
+        {
+            "key": "exec",
+            "value": "nodejs:6"
+        }
+    ],
+    "limits": {
+        "timeout": 60000,
+        "memory": 256,
+        "logs": 10
+    },
+    "publish": false
+}
+```
+
+### Getting an action URL
+
+An action can be invoked through the REST interface via an HTTPS request. To get an action URL, execute the following command:
+
+```
+wsk action get actionName --url
+```
+```
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/actionName
+```
+
+**Note:** Authentication must be provided when invoking an action via an HTTPS request. For more information regarding
+action invocations using the REST interface, see
+[Using REST APIs with OpenWhisk](rest_api.md#actions).
+
+### Saving action code
+
+Code associated with an existing action is fetched and saved locally. Saving is performed on all actions except sequences and docker actions. When saving action code to a file, the code is saved in the current working directory, and the saved file path is displayed.
```

Comment:
"Code associated with an existing action `can be` fetched and saved locally."

Reply:
@mdeuser, I remember @jpkessle saying `is` instead of `can be`. See https://github.com/apache/incubator-openwhisk/pull/2873.

Reply:
lol.  imho, the `is` is a bit misleading as the code is only saved when the `--save` or `--save-as` flag is used. let's see what @jpkessle says.

Reply:
I agree.... +1 for can or may.

Reply:
How about this:

The code for an action may be retrieved and saved locally. Unless a destination is provided, the code is saved to the current working directory and assumes the name of the action. An extension is added to match the action's runtime (e.g., `js` for Node, `jar` for Java, and `zip` for Zip files).

Reply:
@rabbah, a destination can only be provided using the `--save-as` flag.